recipes/vmtk/meta.yaml

```diff
@@ -13,7 +13,9 @@ build:
   number: 0
   skip:
     - True  # [win and py27]
-    - True  # [py37]
+    - True  # [osx and py37]
+    - True  # [linux and py37]
+    - True  # [win and py37]
```

Comment:
You can simply do 

`[(win and py27) or py37]`

Reply:
@beckermr, thank you for the comment. I have tried:

```
- True   # [(win and py27) or py37]
```
```
- True   # [win and py27]
- True   # [py37]
```
```
- True   # [win and py27]
- True   # [osx and py37]
- True   # [linux and py37]
- True   # [win and py37]
```

However, conda-forge still tries to build against 3.7, but not 2.7 for windows. See here for [linux](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=12504&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=288&lineEnd=291&colStart=1&colEnd=38), [osx](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=12504&view=logs&jobId=c572e5d5-7ba7-55ce-5575-7ce1d9f7161e&taskId=5149b458-98d3-59cb-a1a7-f8a51ace7dc8&lineStart=25&lineEnd=28&colStart=1&colEnd=36) and [win](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=12504&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&taskId=23010c01-6a9d-5f94-9037-792618644b77&lineStart=20&lineEnd=22&colStart=1&colEnd=92). Any advice or suggestions will be greatly appreciated!

Reply:
Ahhhh. Sorry. Try this

`[(win and py==27) or py==37]`

I think we have the wrong syntax

Reply:
Thanks, that fixed the py37 problems!